sdk/identity/identity/src/credentials/aggregateCredential.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { TokenCredential, RequestOptionsBase } from "@azure/core-http";
+
+export class AggregateCredential implements TokenCredential {
```

Comment:
I still don't like the name AggregateCredential but I'll defer to Scott's wisdom here.